Crypto/android itest

charlag opened the pull request.

charlag committed:
[android, build] Configure test device via plugin

charlag committed:
[android, build] Use version catalog for tutashared module

charlag committed:
[android, build] Version catalog for calendar

charlag committed:
[android, build] Use version catalog for mail app

charlag committed:
[android, build] Fix deprecations

charlag committed:
[ci] Run Android device tests in CI

vaf-hub commented on .github/workflows/kotlin-test.yml:
```diff
@@ -53,9 +53,25 @@ jobs:
+          api-level: 29
+          arch: x86_64
+          working-directory: ./app-android
+          script: ./gradlew :app:connectedFdroidDebugAndroidTest -PtargetABI=x86_64
```
```suggestion
          script: ./gradlew itest -PtargetABI=x86_64
```
Do you only run it for the app on purpose?

juni2k committed:
[android, build] bump-version: print path of modified gradle script

This should prevent some confusion when "Bumped Android" appears
multiple times in the log because it modified both app and calendar
projects.

juni2k committed:
[android, build] bump-version: mail app's gradle script changed to .kts
